tools/lsp/main.rs

```diff
@@ -176,6 +184,10 @@ struct Cli {
     /// Start the preview in full screen mode
     #[arg(long, action)]
     fullscreen: bool,
+
+    /// Show the preview UI
+    #[arg(long, default_value = "true", action = clap::ArgAction::Set)]
```

Comment:
what's `default_value = "true"` ?   I don't think you need that.

The `fullscreen` option is much simpler

Reply:
imho it shouldn't take arguments.  It should be just `--no-preview-toolbar`

Reply:
or `--no-toolbar`